### ISSUE TITLE
Comments for "Our goal"

### ISSUE THREAD

**Issue by con-cat** [yourbroccoli/exsaasperated-comments#4]
This is the comment thread for [Our goal](https://exsaasperated.com/2020/07/05/our-goal.html) 🥅. Feel free to join in the discussion here 🗿

**Comment by sabbay** [comment]
Love the idea. Keep up the good work! :)

**Comment by ray223** [comment]
Just found your blog. I'm from Orange NSW, ex software developer looking to do exactly the same thing! Can't wait to read along with your progress.  Particularly appreciate your humble goals, user privacy focus and openness :)

**Comment by jkettmann** [comment]
As you said a focus on privacy and using GA are contradictory. Since [friends don't let friends use Google Analytics](https://plausible.io/blog/remove-google-analytics) I'd recommend Plausible.io. Very friendly folks who respect privacy. I use it myself and am happy with it. Did take me literally 5 min to make the switch

**Comment by toothbrush** [comment]
You're right @jkettmann – this is something we've been thinking about a lot. This issue has been fixed in 
https://github.com/yourbroccoli/yourbroccoli.github.io/commit/67cd9d6d124d0be9ff632614725f6a85796ec01b.

**Comment by jkettmann** [comment]
Nice :+1:

**Comment by immanuelfodor** [comment]
It was a good reading, best of luck to you! However, I don't really get what will be the business, the SasS product :) Is it part of the plan to uncover the details in a later blog post or should/will the community provide the idea what you will develop? (CDD, community driven development :D )
